hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassAlertsServiceImpl.java

```diff
@@ -947,6 +1010,49 @@ private void filterByStatuses(String tenantId, AlertsCriteria criteria, Collecti
         if (isEmpty(tenantId)) {
             throw new IllegalArgumentException("TenantId must be not null");
         }
+        List<Event> events = getEvents(tenantId, criteria);
```

Comment:
Same chaining comment.  I can see that this has a little less overhead, but chaining is nice because it consolidates code and reduces potential change instances.